man/docker-build.1.md

```diff
@@ -171,8 +171,8 @@ instruction into the specified target.
 
 ## Building an image and naming that image
 
-A good practice is to give a name to the image you are building. There are
-no hard rules here but it is best to give the names consideration. 
+A good practice is to give a name to the image you are building. Note that 
+only a-z0-9-_. should be used for consistency.  There are no hard rules here but it is best to give the names consideration. 
```

Comment:
> There are no hard rules here but it is best to give the names consideration. 

This is not true, but we've done a lot of work locking this down in the future. A grammar is being added with https://github.com/docker/distribution/issues/690, along with a common package to handle this.